docs/gettingstarted/quickstart.rst

```diff
@@ -0,0 +1,293 @@
+      | Uses application and deployment configuration files                      |
+      +--------------------------------------------------------------------------+
+      | Deploys containers to a cluster of servers                               |
+      +--------------------------------------------------------------------------+
+
+      .. empty-div:: arrow-down center-block hidden-xs hidden-sm
+
+   .. parallel::
+
+      .. mobile-label::
+
+         Live
+
+      .. image:: images/nodes.png
+         :class: center-block offset-top img-responsive
+         :alt: Flocker Node diagram
+
+      Flocker Node
+      ------------
+
+      +--------------------------------------------------------------------------+
+      | Runs on each server in a cluster                                         |
+      +--------------------------------------------------------------------------+
+      | Links, ports and volumes work across hosts                               |
+      +--------------------------------------------------------------------------+
+      | After deployment, containers can move around                             |
+      +--------------------------------------------------------------------------+
+
+      .. empty-div:: arrow-down center-block
+
+   .. parallel::
+
+      Installation
+      ------------
+
+      .. mobile-label::
+
+         Local
+
+      .. noscript-content::
+      
+         OS X
+         ^^^^
+
+         Install the flocker-cli client on your Mac (requires Homebrew):
+
+         .. code-block:: console
+
+            you@laptop:~$ brew update && \
+              brew tap clusterhq/flocker && \
+              brew install flocker-0.3.2
+            
+      .. noscript-content::
+
+         Ubuntu / Debian
+         ^^^^^^^^^^^^^^^
+
+         Install the flocker-cli client on your Linux machine:
+         
+         .. code-block:: console
+         
+            you@laptop:~$ sudo apt-get update && apt-get install -y gcc python2.7 python-virtualenv python2.7-dev && \         
+              virtualenv flocker-tutorial && \
+              flocker-tutorial/bin/pip install --upgrade pip && \
+              flocker-tutorial/bin/pip install --quiet https://storage.googleapis.com/archive.clusterhq.com/downloads/flocker/Flocker-0.3.2-py2-none-any.whl && source flocker-tutorial/bin/activate
+
+         Fedora 20
+         ^^^^^^^^^
+
+         Install the flocker-cli client on your Linux machine:
+         
+         .. code-block:: console
+         
+            you@laptop:~$ sudo yum install -y @buildsys-build python python-devel python-virtualenv && \
+              virtualenv flocker-tutorial && \
+              flocker-tutorial/bin/pip install --upgrade pip && \
+              flocker-tutorial/bin/pip install --quiet https://storage.googleapis.com/archive.clusterhq.com/downloads/flocker/Flocker-0.3.2-py2-none-any.whl && source flocker-tutorial/bin/activate
```

Comment:
Please file a ticket for adding automated test coverage for these two command blocks.